skfda/exploratory/visualization/representation.py

```diff
@@ -778,3 +780,339 @@ def set_color_dict(
     """
     if sample_colors is not None:
         color_dict["color"] = sample_colors[ind]
+
+
+class MixedDataPlot(BasePlot):
+    """
+    Class used to plot a Mixed Data object represented in DataFrames.
+
+    This class visualizes mixed-type datasets where each column of the
+    DataFrame represents a different variable, which can be either functional
+    data (`FData`) or classical numerical data (scalars). Each row
+    is treated as a separate sample.
+
+    Functional variables are plotted using their native `.plot()` method.
+    Vector valued functional data generate multiple subplots, one for each
+    component. Numerical data are shown as scatter plots.
+
+    Supports grouping samples by color using `group`, with optional custom
+    colors (`group_colors`) and labels (`group_names`) for legend display.
+
+    Args:
+        df: pd.DataFrame object that we want to plot.
+        chart: figure over
+            with the graphs are plotted or axis over where the graphs are
+            plotted. If None and ax is also None, the figure is
```

Comment:
Format arguments and code in monospace. E.g.:

```suggestion
            plotted. If ``None`` and ``ax`` is also ``None``, the figure is
```